includes/install.php

```diff
@@ -100,6 +100,12 @@ function give_run_install() {
 
 	flush_rewrite_rules();
 
+	// Create the donor databases.
+	$donors_db = new Give_DB_Donors();
```

Comment:
@mehul0810 can you explain this code

Reply:
@ravinderk I have not added this code. It has appeared always during resolving conflicts so i have kept it as it is considering that this code is creating tables.